modules/serverless-hotfix-patch.adoc

```diff
@@ -0,0 +1,166 @@
+// Module included in the following assemblies:
+//
+// * install/serverless-upgrades.adoc
+
+:_content-type: CONCEPT
+[id="serverless-hotfix-patch_{context}"]
+= Serverless operator maintenance release upgrades
+
+== Latest and maintenance releases
+
+Beginning with {ServerlessProductName} 1.29, the different product versions are available as follows:
+
+* The latest release is available through the `stable` channel.
+* The maintenance release is available through its version-based channel, for example, `stable-1.29`. The maintenance release is the release prior to the latest release, for example if `stable` is 1.30, then maintenance release is `stable-1.29`.
```

Comment:
```suggestion
* The maintenance release is available through its version-based channel, for example, `stable-1.29`. The maintenance release is the release prior to the latest release, for example if `stable` channel includes 1.30, then maintenance release will be available in `stable-1.29` channel.
```

Reply:
It seems like we're calling "stable-1.29" a release. But it is the channel where the release is available. Just a little wording issue.